docs/tutorial/testing-contracts.md

```diff
@@ -217,7 +217,7 @@ describe("Token contract", function () {
 
       // Check balances.
       const finalOwnerBalance = await hardhatToken.balanceOf(owner.address);
-      expect(finalOwnerBalance).to.equal(initialOwnerBalance - 150);
+      expect(finalOwnerBalance).to.equal(initialOwnerBalance.sub(150));
```

Comment:
```suggestion
      expect(finalOwnerBalance).to.equal(initialOwnerBalance.subn(150));
```

Reply:
Can you elaborate? `subn` doesn't exist in ethers API.

Reply:
Sorry, I was confusing it with [the BN.js used by ethereums](https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/util#re-exports). I retract my suggestion :)